package.json

```diff
@@ -31,29 +31,40 @@
     "appium-android-driver": "^1.10.27",
     "appium-base-driver": "^2.0.18",
     "appium-logger": "^2.1.0",
-    "appium-uiautomator2-server-test-1": "0.0.53",
+    "appium-uiautomator2-server-test-1": "0.0.54",
     "appium-support": "^2.0.9",
     "asyncbox": "^2.3.1",
+    "bluebird": "^2.10.2",
     "babel-runtime": "=5.8.24",
     "lodash": "^3.10.1",
+    "request-promise": "^4.1.1",
     "source-map-support": "^0.3.2",
-    "utf7": "^1.0.0",
-    "yargs": "^3.29.0",
     "teen_process": "^1.3.1",
-    "request-promise": "^4.1.1",
-    "bluebird": "^2.10.2"
+    "utf7": "^1.0.0",
+    "yargs": "^3.29.0"
   },
   "scripts": {
     "install": "node ./bin/install.js",
     "prepublish": "./node_modules/.bin/gulp prepublish",
     "test": "./node_modules/.bin/gulp once",
-    "watch": "./node_modules/.bin/gulp"
+    "watch": "./node_modules/.bin/gulp",
+    "lint": "gulp eslint"
```

Comment:
I personally like to have a message printed out here, to make it clear what is happening in pre-commit.

Like https://github.com/appium/appium-xcuitest-driver/blob/master/package.json#L53 and https://github.com/appium/appium-xcuitest-driver/blob/master/package.json#L57